packages/vscode-extension/src/telemetry/extTelemetry.ts

```diff
@@ -35,7 +35,7 @@ export namespace ExtTelemetry {
       case Stage.create:
         return TelemetryEvent.CreateProject;
       case Stage.update:
-        return TelemetryEvent.UpdateProject;
+        return TelemetryEvent.AddResource;
```

Comment:
why change this?

Reply:
Just keep the name consistent with the meaning